src/encoded/audit/analysis.py

```diff
@@ -115,13 +376,23 @@ def audit_experiment_standards_dispatcher(value, system, files_structure):
         'ENCODE2-Mouse',
```

Comment:
I'm wondering for `len(value['pipeline_award_rfas']) != 1` above, should analysis having mixed rfas be audited?

Reply:
I think having a mixed rfas audit sounds good, but maybe I can add that in a separate ticket to avoid complicating the QA.
After reviewing what we have on the portal, I did find 472 cases of > 1 rfas, but all of them were Graveley lab analyses. So at least, I don't think we are wrongly excluding any ENCODE uniformly processed pipelines with this check.